docs/rl-algorithms/ppo.md

```diff
@@ -88,12 +110,14 @@ Learning curves:
 
 Tracked experiments and game play videos:
```

Comment:
nit, if it's classic control env, then probably use videos instead game play videos.